pattern-1/bosh-release/jobs/wso2apim_analytics/spec

```diff
@@ -33,16 +37,16 @@ properties:
   wso2apim_analytics.address:
     description: wso2apim address
 
-  wso2apim_analytics.mysql.hostname:
+  wso2apim.mysql.hostname:
```

Comment:
In BOSH, providing default values could be avoided and defining values for the properties can be made mandatory (e.g. check properties defined [here](https://github.com/wso2/pivotal-cf-is/blob/master/pattern-2/bosh-release/jobs/wso2is_1/spec#L20)).

Further, IP Address `192.168.50.1` was used earlier when testing the BOSH Release with BOSH-Lite. It is better if we can force the user to define the IP Address (a mandatory property) rather than providing an IP Address which is specific only to a `dev` environment.